requirements.txt

```diff
@@ -5,7 +5,7 @@ pysam==0.20.0
 sqlalchemy==1.4.44
 connexion==2.14.1
 MarkupSafe==2.1.1
-candigv2-authx@git+https://github.com/CanDIG/candigv2-authx.git@v2.0.0
+candigv2-authx@git+https://github.com/CanDIG/candigv2-authx.git@daisieh/query-token
```

Comment:
Should this be `develop` after the changes get merged?

Reply:
Yep, I'll up the version for authx and then change this value in all of the PRs.